source/MaterialXCore/Definition.cpp

```diff
@@ -37,6 +38,44 @@ InterfaceElementPtr NodeDef::getImplementation(const string& target) const
     return InterfaceElementPtr();
 }
 
+InterfaceElementPtr NodeDef::getImplementation(const string& language, const string& target) const
```

Comment:
Can this be combined with the existing NodeDef::getImplementation above, where language simply falls back to the empty string if left unspecified?